hack/build-images.sh

```diff
@@ -58,10 +66,17 @@ if [ "$verbose" = true ]; then
 fi
 
 echo "Building image ${prefix}metrics-hawkular-metrics:${version}"
-docker build -t "${prefix}metrics-hawkular-metrics:${version}"       ../hawkular-metrics/
-docker build -t "${prefix}metrics-cassandra:${version}" ../cassandra/
-docker build -t "${prefix}metrics-heapster:${version}"        ../heapster/
-docker build -t "${prefix}metrics-deployer:${version}"    ../deployer/
+if [ "$no_cache" = true ]; then
```

Comment:
In the prior step, couldn't you just set  '--no-cache' as an env variable or have the variable empty.  Then Lines 70-73 could use that variable and it would either build with caching or not, thus removing the need to duplicate these lines